### PR TITLE
[Ex2] implement part of llvm visitor

### PR DESCRIPTION
**Pull request by chenmoryosef**

### CODE REVIEW

File: src/ast/AstLlvmPrintVisitor.java [hunk 1]
```diff
@@ -212,86 +233,258 @@ public void visit(AssignArrayStatement assignArrayStatement) {
...
-        // It means we
-        if (e.e1().getClass() == IntegerLiteralExpr.class )
-        {
-
-        }
-        else {
-            e.e1().accept(this);
-        }
-        visitBinaryExpr(e, "+");;
+        visitBinaryExpr(e, "add");
```
Comment (galiheim) [inline src/ast/AstLlvmPrintVisitor.java:248]: awsome<3

File: src/ast/AstLlvmPrintVisitor.java [hunk 2]
```diff
@@ -212,86 +233,258 @@ public void visit(AssignArrayStatement assignArrayStatement) {
...
     @Override
     public void visit(SubtractExpr e) {
-        visitBinaryExpr(e, "-");
+        visitBinaryExpr(e, "sub");
     }
 
     @Override
     public void visit(MultExpr e) {
-        visitBinaryExpr(e, "*");
+        visitBinaryExpr(e, "mul");
```
Comment (galiheim) [inline src/ast/AstLlvmPrintVisitor.java:258]: looks great fire on u

**Review by galiheim — APPROVED** [review]
looks great!!

**Review by rachelrobins — APPROVED** [review]
Good job!

**Pull request merged**